code/drasil-example/glassbr/lib/Drasil/GlassBR/ModuleDefs.hs

```diff
@@ -159,7 +163,7 @@ extractColumnCT = funcDef "extractColumn" "Extracts a column from a 2D matrix"
   ]
 
 interpY :: Func
-interpY = funcDef "interpY" 
+interpY = funcDef (showUID U.interpY)
```

Comment:
Surely there's another field than `UID` in `U.interpY` which also have value "interpY" that could be used? Of this whole change, this is the one bit that sticks out as a hack, all the rest seems great.

Reply:
@JacquesCarette @balacij  I have made the requested changes in 17d3425547ecceaa52a4d347860ab93713b26c01

Reply:
You have - but the changes are too "brute force". You most definitely should not be importing HughesPJ into any example! The right thing to do is to write an appropriate function (that does what you just coded up) somewhere in an appropriate Drasil package, and then call it here.  Figuring out the right signature is going to be a fun design problem.

Reply:
@JacquesCarette would a showSymb-like function be helpful elsewhere as well (not sure of which package to place into yet)?

```
showSymb :: Symbol -> String
showSymb a = render $ symbolDoc a
```

This would allow for the removal of HughesPJ from the example

```
interpZ :: Func
interpZ = funcDef (showSymb $ symbol U.interpZ Implementation)
```

Reply:
Yes, such a function would be good. Probably belongs in drasil-printers. I'd be tempted to put it under the ASCII printer.

Reply:
@JacquesCarette I had just edited the above comment to suggest placing it in Drasil/Symbol/Helpers.hs, but I will look into placing it into drasil-printers.

Reply:
@JacquesCarette The requested change to add a new function has been applied in the latest commit. The PR is updated.